styles/styles.css

```
@@ -0,0 +1,291 @@
+@font-face {
+    font-family: "Rubik";
+    font-style: normal;
+    font-weight: 400;
+    src: url("../fonts/rubik/rubik-400.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "Rubik";
+    font-style: normal;
+    font-weight: 700;
+    src: url("../fonts/rubik/rubik-700.woff2") format("woff2");
+    font-display: swap;
+}
+
+@font-face {
+    font-family: "Raleway";
+    font-style: normal;
+    font-weight: 800;
+    src: url("../fonts/raleway/raleway-800.woff2") format("woff2");
+    font-display: swap;
+}
+
+html {
+    height: 100%;
+}
+
+body {
+    margin: 0;
+    display: flex;
+    flex-direction: column;
+     min-height: 100%;
+    font-family: "Rubik", "Arial", sans-serif;
+    font-style: normal;
+    font-size: 18px;
+    line-height: 30px;
+    background-color:#F0F0F0;
+    color: #000000;
+}
+
+.page-header {
+    background-color: #ffe17f;
+    color: #000000;
+}
+
+.user-navigation {
+    display: flex;
+    flex-direction: column;
+    width: 1160px;
+    margin: 0 auto;
+   
+}
+
+.header-logo {
+    width: 145px;
+    margin: 0;
+    padding: 0;
+}
+
+.form-search {
+    width: 517px;
+}
+
+.search {
+    font-size: 16px;
+    line-height: 20px;
+}
+
+.navigation-link-login {
+    width: 231px;
+}
+
+.comparison-shopping-cart {
+    width: 240px;
+    margin-left: auto;
+    
+    
+}
```

```suggestion
}
```
держи стили в чистоте